Search/bfsnew.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+using namespace std;
+int queue[5];
+int a[5];
+int root;
+int L;
+int R;
+int front=0;
+int end=0;
+void BFS();
+void push();
+void pop();
+
+int main()
+{
+	for(int i=0;i<5;i++)
+	{
+		cin>>a[i];
```

Comment:
Graph is a pair of _set of vertices & edges_.
What does **a** store? Vertices or Edges??
You should take both _(vertices & edges)_ as input.